src/services/user.services.ts

```diff
@@ -0,0 +1,14 @@
+import { db } from '../../prisma/db';
+import { UserType } from '../types/user';
+
```

Comment:
You can change the service as a static class to later use:

UsersService.getUsers();

The advantage of this is that it aids code comprehension, and being static means you don't have to instantiate the class

Reply:
Fixed